admin/admin.go

```diff
@@ -26,11 +27,17 @@ type Options struct {
 }
 
 type Service struct {
-	DB               database.DB
-	ProvisionerSet   map[string]provisioner.Provisioner
-	Email            *email.Client
-	Github           Github
-	AI               ai.Client
+	DB             database.DB
+	ProvisionerSet map[string]provisioner.Provisioner
+	Email          *email.Client
+	Github         Github
+	AI             ai.Client
+	// Assets as reduced surface of storage.BucketHandle
+	// enables us to use a mock in tests.
+	Assets interface {
+		SignedURL(object string, opts *storage.SignedURLOptions) (string, error)
+		Object(path string) *storage.ObjectHandle
+	}
```

Comment:
If you don't actually mock it in the test, let's just keep this as `*storage.BucketHandle`. If you add mocks, we should use a separate interface instead of an inline interface here.